go-client/examples/input_table/main.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/apache/arrow/go/v8/arrow"
+	"github.com/deephaven/deephaven-core/go-client/client"
+	"github.com/deephaven/deephaven-core/go-client/examples/common"
+)
+
+// This example shows how to use Input Tables.
+// Though Deephaven has direct support for streaming data through Apache Kafka,
+// Input Tables make are a generic interface for streaming data from any source.
```

Comment:
See all comments in `query/main.go` and apply the concepts to this file.

e.g. a reader is going to wonder why Kafka got dragged into an example on another topic.  The comment is a distraction from the story you are trying to get across with the example.